services/web/server/src/simcore_service_webserver/announcements/_models.py

```diff
@@ -0,0 +1,54 @@
+from datetime import datetime
+from typing import Any, ClassVar, Literal
+
+from pydantic import BaseModel, validator
+
+
+# NOTE: this model is used for BOTH
+# - parse+validate from redis
```

Comment:
Should we then also somehow validate that e.g. the product is truly present in the postgres? Otherwise what will happen, will the announcement just not be displayed?

Reply:
This version will not list invalid announcements, yes